### PR TITLE
HPCC-17317 DOCS: ECLWatch Add Delete Part

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on 
using the Roxie Package Maps, Add, Delete, Activate Part.

Initial Pull Request, Expecting comments and feedback. 

@JamesDeFabia please review
@miguelvazq please review
@afishbeck please review 

Build results and PDF can be reviewed at 
http://10.240.32.243/job/DocDev0IV/3/CE=ubuntu-14.04-amd64-1/artifact/build-docs/Release/docs/The_ECL_Watch_Manual-6.3.0-trunk0.pdf
(Chrome users may need to download the file to review)

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [x] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [x] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 1]
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
         package map. To examine a package map, select a package map from the
         list.</para>
 
-        <para>You can select the <emphasis role="bold">Validate Package
-        Map</emphasis> tab to validate a package map.</para>
+        <sect3>
+          <title id="Packagemap_Actions">Packagemap Actions</title>
```
Comment (JamesDeFabia) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: s/b two words: Package Map Actions

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 2]
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
...
+          <para><figure>
+              <title>Add Package Maps</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWA470A.jpg"
+                             vendor="ECLWatchSS" />
+                </imageobject>
+              </mediaobject>
+            </figure>To Add a packagemap to the target cluster:</para>
```
Comment (JamesDeFabia) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: package map (two words)

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 3]
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
...
+            </listitem>
+
+            <listitem>
+              <para>Choose the <emphasis role="bold">Target </emphasis>to
+              associate the package map with.</para>
+            </listitem>
+
+            <listitem>
+              <para>Choose a Process FIlter /*what is command line
+              equiv?*/</para>
```
Comment (JamesDeFabia) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: You left a comment. Replace with the description

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 4]
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
...
+
+            <listitem>
+              <para>Enter the IP address or hostname of the remote Dali to use
+              for logical file lookups for the <emphasis role="bold">Remote
+              Dali IP Address</emphasis> field.</para>
+            </listitem>
+
+            <listitem>
+              <para>Check the boxes to Activate or Overwrite as
+              desired.</para>
```
Comment (JamesDeFabia) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: Check the boxes to Activate or Overwrite, as desired

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 5]
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
...
 
-        <para>Press the <emphasis role="bold">Validate</emphasis> button to
-        validate the package map. The result is shown on the <emphasis
-        role="bold">Validate Active Package Map</emphasis> tab. You can also
-        Validate the package content, on the V<emphasis role="bold">alidate
-        Package Content</emphasis> tab.</para>
+          <para><emphasis role="bold">Deactivate Package Map</emphasis></para>
+
+          <para>Press the deactivate button to deactivate the currently active
+          package map.</para>
```
Comment (JamesDeFabia) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: `<emphasis role="bold">Deactivate</emphasis>  button`

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 6]
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
...
+            </listitem>
+          </orderedlist>
+
+          <para><emphasis role="bold">Activate Package Map</emphasis></para>
 
-        <para>Choose the <emphasis role="bold">Target</emphasis> and <emphasis
-        role="bold">Process</emphasis> from the drop lists on the Validate
-        Package Map tab.</para>
+          <para>Press the Activate button to deactivate the currently active
+          package map and make the selected package map active.</para>
```
Comment (JamesDeFabia) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: `<emphasis role="bold">Activate</emphasis>  button`

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Comments inline

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml (hunk 1 above)
Comment (g-pan) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: Don't think you are looking at the latest commit. This was fixed in a later commit.

**Comment by miguelvazq** [comment]
@g-pan Additional functionality for parts is located in the details as we discussed offline.

**Comment by richardkchapman** [comment]
Needs rebasing foe a clean merge

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 7]
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
...
-        <para>You can select the <emphasis role="bold">Validate Package
-        Map</emphasis> tab to validate a package map.</para>
+        <sect3>
+          <title id="Packagemap_Actions">Packagemap Actions</title>
 
-        <para><figure>
-            <title>Validate Package Maps</title>
+          <para>To update package mappings, you would edit the packagemap file
+          (or a copy) add and activate the new one, and then delete the old
+          one. You can do this through the interface in ECLWatch.</para>
```
Comment (afishbeck) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: ECLWatch or command line?

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml (hunk 3 above)
Comment (afishbeck) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: btw, I don't think I support process filter on command line.  It's one of those complex things that if I start to think people really want or need it I will add but over the last 5 years or so no one seems to be needing it.

**Review by afishbeck — COMMENTED** [review]
@g-pan a few comments and general bits of knowledge.

**Comment by richardkchapman** [comment]
Needs rebasing for a clean merge

**Comment by g-pan** [comment]
Keeping pull request open, while work in progress - preserving comments.  Will rebase when ready. Please hold off merging until rebased.

**Comment by g-pan** [comment]
Due in part to a hardware outage, and in part to some of the commit conflicts and comments here, I am closing this pull request, only to replace it with a single commit pull request.

**Pull request closed without merging**